scripts/stats.php

```diff
@@ -65,10 +65,10 @@
    <form action="views.php" method="GET">
     <input type="hidden" name="sort" value="<?php if(isset($_GET['sort'])){echo $_GET['sort'];}?>">
       <input type="hidden" name="view" value="Species Stats">
-      <button <?php if(!isset($_GET['sort']) || $_GET['sort'] == "alphabetical"){ echo "style='background:#9fe29b !important;'"; }?> class="sortbutton" type="submit" name="sort" value="alphabetical">
+      <button <?php if(!isset($_GET['sort']) || $_GET['sort'] == "alphabetical"){ echo "class='active !important;'"; }?> class="sortbutton" type="submit" name="sort" value="alphabetical">
```

Comment:
Line 68 & 71: Recommendation to put the class="sortbutton" in the <div> on line 63, to avoid weird cutoff when hovering over the sortbuttons.

Reply:
OK added thanks